### ISSUE TITLE
Allow for private methods as context?

### ISSUE THREAD

**Issue by brendon** [palkan/action_policy#20]
Hi @palkan :) In my application `current_user` was a private method in `ApplicationController`. I'm just wondering why you're using `public_send`? I've made `current_user` a public method for now but I always thought it was good practice to not have public methods in controllers as they can become actions?

https://github.com/palkan/action_policy/blob/124b62b1851b14605f4be41281f53c1feb4d54cf/lib/action_policy/behaviour.rb#L53

**Comment by palkan** [comment]
>  I'm just wondering why you're using public_send

Don't know))

Agree, It make sense to just use `send` here. Would you like to propose a PR?)

**Comment by brendon** [comment]
Will do. Though today I learned that `hide_action` and crew were removed from Rails 5 because they hark back to the old days before RESTful controllers, where we used to pass through `:controller/:action/:id` *shudder*, and that it's not really necessary to put non-action methods in the `private` or `protected` space though I still think it's a good discipline.

Still it's helpful to be able to get the `current_user` even if it's private.
